molecule/builder/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:14.04
+
+RUN apt-get -y update && apt-get install -y python rsync sudo bash \
+            devscripts git aptitude ipython libssl-dev ntp \
+            python-dev python-pip python-ipdb ruby tmux vim paxctl && \
+            aptitude -y dist-upgrade && \
+            apt-get clean
```

Comment:
Bummer to have these hard-coded rather than vars-based, but that's Docker for ya. Worth adding some config tests to track these explicitly, since many of them were present implicitly in the Ubuntu Trusty build VM.